GiniVision/Classes/Core/GiniScreenAPICoordinator+Review.swift

```diff
@@ -94,13 +94,12 @@ extension GiniScreenAPICoordinator: MultipageReviewViewControllerDelegate {
         }
     }
     
-    func refreshMultipageReview(with documentRequests: [DocumentRequest]) {
+    func refreshMultipageReviewNextButton(with documentRequests: [DocumentRequest]) {
```

Comment:
I would have been ok with the old name. It doesn't have to specify everything it does. It's just a method for updating the UI. The specifics can be encapsulated.

Reply:
Before it made sense, because it did different things. But know it is just the next button refreshing.